modules/maven/s2i/artifacts/opt/jboss/container/maven/s2i/maven-s2i

```diff
@@ -1,7 +1,7 @@
 
 # include dependencies
 source "${JBOSS_CONTAINER_UTIL_LOGGING_MODULE}/logging.sh"
-source "${JBOSS_CONTAINER_S2I_CORE_MODULE}/s2i-core"
```

Comment:
There is an implicit dependency on the `jboss.container.java.s2i` module.
We can't depend on `jboss.container.java.s2i` module that implements the s2i logic, WildFly has its own different s2i logic.
We need the core to be clean from java.s2i specifics.